packages/wonder-blocks-core/components/id-provider.md

```diff
@@ -0,0 +1,18 @@
+### Example: Usage with form fields
+
+This example allows you to generate an unique ID and make it available to associate the `<label>` and `<input>` elements. To see this example in action, check that `label[for]` and `input[id]` are using the same id.
+
+```jsx
+const children = (uniqueId) => (
```

Comment:
It might be slightly more idiomatic to put this inline below, rather than having `children` be a separate variable. So something like:

```
<IDProvider scope="field">
    {(uniqueId) => (
        <label htmlFor={uniqueId}>
            ...
        </label>
    )}
</IDProvider>
```

Reply:
Makes total sense, I will do it right away